server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/deployment/ServiceHandler.java

```diff
@@ -49,7 +49,6 @@ private void readProviderServices() throws IOException {
         if (application.getProvider() != null && !application.getServices().isEmpty() && application.getConnection() != null) {
```

Comment:
Same as above, consider using `CollectionUtils.isEmpty()`.

Reply:
I think here not possible because these are no lists.